### PR TITLE
[IMP] accounting/l10n: table with list of modules to install Luxembourg

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

**Comment by jcs-odoo** [comment]
@robodoo r+

File: content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst [hunk 1]
```diff
@@ -8,9 +8,21 @@ Configuration
 :ref:`Install <general/install>` the following modules to get all the features of the Luxembourgish
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst:8]:
```suggestion
:ref:`Install <general/install>` the following modules to get all features of the Luxembourgish
```

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
No issues as always, just an extra "the".

File: content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst (hunk 1 above)
Comment (jcs-odoo) [reply content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst:8]: Ah okay, good to know. It's not part of this commit and this text is already in several docs in several versions so I'll remove this "the" with another PR. thanks :)

Comment (jcs-odoo) [reply content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst:8]: @toaa-odoo

**Pull request closed without merging**